content/millennium/r4/request-and-response/communication.md

```diff
@@ -59,7 +58,7 @@ _Implementation Notes_
 <%= authorization_types(practitioner: true, patient: false, system: true) %>
 
 ### Parameters
-
+ 
```

Comment:
nit: extra space

Reply:
Rebased with Yuyu's, and the space went away: 
No changes: https://github.com/cerner/fhir.cerner.com/pull/435/files#diff-a46637e8cbba5ebbd797adadefc1bc2eR59-R61
Yuyu's: https://github.com/cerner/fhir.cerner.com/pull/433/files#diff-a46637e8cbba5ebbd797adadefc1bc2eR60-R61